internal/cloudapi/v2/v2_koji_test.go

```diff
@@ -553,7 +552,7 @@ func TestKojiRequest(t *testing.T) {
 	handler.ServeHTTP(rec, req)
 	resp := rec.Result()
 
-	var status api.Status
+	var status interface{}
```

Comment:
nitpick: Shouldn't this be `ComposeStatus` instead of `interface{}`? Nevertheless I can not think of any situation when using `interface{}` in this unit test would cause an issue or cover a potential bug.

Reply:
I prepended a commit that removes this test while making sure that everything is still covered (in even better way).